crates/db_views_actor/src/person_view.rs

```diff
@@ -94,41 +94,39 @@ impl<'a> PersonQuery<'a> {
         .or_filter(person::display_name.ilike(searcher));
     }
 
-    query = match self.sort.unwrap_or(SortType::Hot) {
-      SortType::New | SortType::NewComments => query.order_by(person::published.desc()),
-      SortType::Old => query.order_by(person::published.asc()),
-      SortType::Hot | SortType::Active | SortType::TopAll => {
-        query.order_by(person_aggregates::comment_score.desc())
-      }
-      SortType::MostComments => query.order_by(person_aggregates::comment_count.desc()),
-      SortType::TopYear => query
+    query = match self.sort.unwrap_or(PersonSortType::TopAll) {
+      PersonSortType::New => query.order_by(person::published.desc()),
+      PersonSortType::Old => query.order_by(person::published.asc()),
+      PersonSortType::MostComments => query.order_by(person_aggregates::comment_count.desc()),
+      PersonSortType::TopAll => query.order_by(person_aggregates::comment_score.desc()),
+      PersonSortType::TopYear => query
         .filter(person::published.gt(now - 1.years()))
         .order_by(person_aggregates::comment_score.desc()),
-      SortType::TopMonth => query
+      PersonSortType::TopMonth => query
```

Comment:
I don't know if a lot of these ones make sense. This is filtering by persons created (published) within the last month.

Maybe just `CommentScore` instead of all these `Top` sorts.